engine/audio/src/audiocapture.cpp

```diff
@@ -18,6 +18,8 @@
   limitations under the License.
 */
 
+#include <stdio.h>
```

Comment:
I added this temporarily so I could use printf for hacky debug

Reply:
No need. You can use this way to print on console with Qt:
qDebug() << "Something" << var;